src/hb-font.cc

```diff
@@ -1366,6 +1398,51 @@ hb_font_get_glyph_shape (hb_font_t *font,
   font->get_glyph_shape (glyph, dfuncs, draw_data);
 }
 
+/**
+ * hb_font_draw_glyph:
+ * @font: #hb_font_t to work upon
+ * @glyph: : The glyph ID
+ * @dfuncs: #hb_draw_funcs_t to draw to
+ * @draw_data: User data to pass to draw callbacks
+ *
+ * Alternative name for hb_font_get_glyph_shape()
+ *
+ * Since: Replaceme
+ */
+void
+hb_font_draw_glyph (hb_font_t *font,
+		    hb_codepoint_t glyph,
+		    hb_draw_funcs_t *dfuncs, void *draw_data)
+{
+  hb_font_get_glyph_shape (font, glyph, dfuncs, draw_data);
+}
```

Comment:
I’m a bit uneasy about this naming, since the function is not doing any drawing, but calling client callbacks that will do the actual drawing. If we grow a rasterization API at some point, this naming is going to be confusing.